pulp_2to3_migration/app/tasks/migrate.py

```diff
@@ -33,6 +36,52 @@ def migrate_from_pulp2(migration_plan_pk, validate=False, dry_run=False):
         validate (bool): If True, don't migrate unless validation is successful.
         dry_run (bool): If True, nothing is migrated, only validation happens.
     """
+
+    def get_repo_types(plan):
```

Comment:
Maybe this should be done a s a part of plan processing...
Thoughts?
@ipanova , @dralley